### DISCUSSION TITLE
Any email notification when schedule back ups fails

### DISCUSSION THREAD

**Discussion by mcpdeveloper443** [ravendb/ravendb#12001] [Ideas]
Hi ,

If worse case any schedule back up failed to take back up, is any email prompt notification to user ?

**Comment by ayende** [comment]
No, the usual way you'll monitor that is through the alerts system.
The idea is that you'll plug your monitoring solution to the alerts exposed by RavenDB

**Reply by mcpdeveloper443** [reply]
monitor solution provided from RavenDB ? please help to provide some more inputs on this. How to plug monitoring solution.

**Reply by ayende** [reply]
You can use the SNMP monitoring, yes. See: https://ravendb.net/docs/article-page/5.0/Csharp/server/administration/SNMP/snmp
